### ISSUE TITLE
A Capital City

### ISSUE THREAD

**Issue by TRManderson** [lsenjov/politico#12]
Many of the ruling party have already expressed concern over where we will locate the capital of our great nation, and this is something that must be discussed. Does anyone wish to propose a location?

**Comment by TRManderson** [comment]
I think a good criteria for city selection is high GDP (I don't think per capita is the best choice honestly, so GDP total), high population, or a respectable balance of the two. To this end, three cities seem like good choices:
- Raanan, highest GDP
- Gawahir, highest population
- Haluk, 2nd highest GDP, 2nd highest population

Notable mentions go to Pons for highest per-capita GDP and Heiner for having a GDP very very close to that of Haluk.

**Comment by rslabbert** [comment]
Since the capital will be the focus of the nation, both from the outside, and the inside, I'm in favour of a balance since that will prevent the focus of the nation from shifting too much towards a single goal.

Thus I am proposing the capital to be in Haluk.

**Comment by TRManderson** [comment]
I think that using a measure of total GDP already sufficiently accounts for the value of a high-population city (by virtue of multiplication), so would prefer Raanan or Haluk over Gawahir. Haluk seems like a reasonable enough choice.

**Comment by Velonere** [comment]
The current cities of [INSERT NAME HERE] represent the resolve of the people who have endured the revolution that has ravaged our great country and no cities resolve was greater than the other thus selecting one of the current cities for the capital will only give rise to hostilities and the rest of the country will hold contempt towards the selected city.

Therefore, I propose the construction of a new Capital City.

The city will be fully planned out before its construction unlike the satellite cities that are our current ones. A planned city has optimised infrastructure and is by far the superior choice.

This proposal will create jobs for the new citizens of [INSERT NAME HERE] and will be a symbol of national pride as everyone has contributed to its construction.

Therefore, I propose the construction of a new Capital City.

**Comment by ghost** [comment]
I agree with Ruben that the capital city should have a balanced, median population and GDP. However, I disagree that Haluk achieves this balance. 

As demonstrated below in the R programming language

```
> a = c(1162770,1890190,1816920,1701480,1410830,1791010)
> median(a)
[1] 1746245
> c = 1701480 + 1791010
> c/2
[1] 1746245
```

the median population happens to be an average of the populations of Heiner and Ranaan. Also, as demonstrated below

```
> b = c(781,593,693,740,849,755)
> median(b)
[1] 747.5
> 755-747
[1] 8
> 740-747
[1] -7
```

the median gdp belongs to Ranaan (755) and Heiner (740) too. Ranaan is a distance of 8 from the median while Heiner is a distance of 7 from the median. Hence I propose the capital to lie in Heiner.
